Strategy11/formidable-forms
# Update Default Form Styling

This PR updates and redesigns the default form fields styling.

## QA URLs:
Frontend URL: https://qa.formidableforms.com/sherv/wp-admin/admin-ajax.php?action=frm_forms_preview&form=new-style-fields

Admin URL: https://qa.formidableforms.com/sherv/wp-admin/admin.php?page=formidable&frm_action=edit&id=71

## Pro Version PR:
https://github.com/Strategy11/formidable-pro/pull/5012

## Testing Instructions:
1. Go to `WP Admin > Formidable > Forms`.
2. Create a new Form and add fields in the Output sections mentioned below.
3. Verify that the updated design is applied correctly without any bugs.

**Note: Make sure to tap the "Reset to Defaults" button to apply the new styles as shown below:**

![CleanShot 2024-05-23 at 14 09 15@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/9be4b912-923e-4300-9ff9-e603bc5f2f5c)

## Output:

### Input, Textarea Elements:
![CleanShot 2024-05-23 at 18 46 52@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/d6d54cfd-382f-47cd-b8da-1eb9eee5ecf6)
![CleanShot 2024-05-23 at 18 59 29@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/9aea8273-5ceb-4d14-a5ae-ae7f648f57fd)

### Select Element:
![CleanShot 2024-05-23 at 18 55 51@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/256b70c0-193e-4766-b4d2-543bf22b0ff8)

### International Phone Element:
![CleanShot 2024-05-23 at 18 58 06@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/c4016a8d-6345-4803-b9ac-64c628960478)

### Checkbox Elements:
![CleanShot 2024-05-23 at 19 06 41@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/30198b84-80b6-449f-9bd6-78424d0f538e)
![CleanShot 2024-05-23 at 19 08 29@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/56a6be8c-666e-4f13-8cb0-e0cda65b5a22)

### Radio Button Elements:
![CleanShot 2024-05-23 at 19 10 07@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/fc4d0fcd-903f-462f-a69f-010905573231)

### Button Elements:
![CleanShot 2024-05-23 at 19 16 25@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/5d15f7ad-a91f-4752-8f74-f61bdd12bfda)

### File Upload Elements:
![CleanShot 2024-05-23 at 19 23 02@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/c95dcc53-0e4b-4ca9-9058-ba501af9618d)
![CleanShot 2024-05-23 at 19 23 36@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/074ad468-9984-45e6-933e-e199ce215c43)
![CleanShot 2024-05-23 at 19 24 02@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/6815ac64-800a-481a-a05b-da461953ed59)

### Calendar Element:
![CleanShot 2024-05-23 at 19 26 06@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/3e94c40e-7590-4c03-bcf5-81b236cd2c4d)

### Slider (Range Input) Elements:
![CleanShot 2024-05-23 at 19 30 52@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/666791f2-11d1-46fd-a81a-d7aeaa92969e)

### Stepper Elements:
![CleanShot 2024-05-23 at 19 32 02@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/274e7ffa-9a2c-477f-bae7-59de58d1704c)
![CleanShot 2024-05-23 at 19 32 16@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/26bbca11-3914-494a-afb3-63ad9b54bd59)

### Star Rating Element:
![CleanShot 2024-05-23 at 19 35 00@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/f564bcd7-e0f6-404b-91bb-3cc4f9beb1af)

### Section Elements:
![CleanShot 2024-05-23 at 19 36 39@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/515f2b08-260d-4baf-ad77-3c78a32cbb97)
![CleanShot 2024-05-23 at 19 35 59@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/1e96683c-e339-45c1-99f3-3f863eebf641)

### Error State:
![CleanShot 2024-05-23 at 19 00 37@2x](https://github.com/Strategy11/formidable-forms/assets/69119241/23eeeda8-0b42-4f35-a0f4-6790caef13ad)
